<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>543</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Tabula Rasa</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Ressources/ressources.qrc">
    <normaloff>:/Icones/icone_Application.png</normaloff>:/Icones/icone_Application.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="actionSauvegarder_sous"/>
    <addaction name="actionImprimer"/>
    <addaction name="separator"/>
    <addaction name="actionExporter_vers_une_image"/>
    <addaction name="actionExporter_en_PDF"/>
    <addaction name="actionExporter_SVG"/>
    <addaction name="separator"/>
    <addaction name="actionFermer_l_onglet"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>&amp;Edition</string>
    </property>
    <addaction name="actionTout_s_lectionner"/>
    <addaction name="actionCopier_la_s_lection"/>
    <addaction name="actionColler_la_s_lection"/>
    <addaction name="actionSupprimer_la_s_lection"/>
    <addaction name="separator"/>
    <addaction name="actionRenommer_l_algorithme"/>
    <addaction name="actionRedimensionner_l_Algorithme"/>
   </widget>
   <widget class="QMenu" name="menuG_n_rer">
    <property name="title">
     <string>&amp;Générer</string>
    </property>
    <addaction name="actionVers_Ada"/>
    <addaction name="actionVers_Java"/>
    <addaction name="actionVers_C_C"/>
   </widget>
   <widget class="QMenu" name="menuA_propos">
    <property name="title">
     <string>&amp;Aide</string>
    </property>
    <addaction name="actionManuel"/>
    <addaction name="separator"/>
    <addaction name="actionA_propos_de_Tabula_Rasa"/>
    <addaction name="actionA_propos_de_Qt"/>
   </widget>
   <widget class="QMenu" name="displayMenu">
    <property name="title">
     <string>Affichage</string>
    </property>
   </widget>
   <addaction name="menu_Fichier"/>
   <addaction name="menuEdition"/>
   <addaction name="displayMenu"/>
   <addaction name="menuG_n_rer"/>
   <addaction name="menuA_propos"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMode_Insertion"/>
   <addaction name="actionMode_Edition"/>
   <addaction name="separator"/>
   <addaction name="actionAction"/>
   <addaction name="actionProcedure"/>
   <addaction name="actionSortie"/>
   <addaction name="separator"/>
   <addaction name="actionIteration"/>
   <addaction name="actionCondition"/>
   <addaction name="actionConditionMultiple"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNouveau"/>
   <addaction name="actionOuvrir"/>
   <addaction name="actionEnregistrer"/>
   <addaction name="actionImprimer"/>
   <addaction name="separator"/>
   <addaction name="actionExporter_vers_une_image"/>
   <addaction name="actionExporter_en_PDF"/>
   <addaction name="actionExporter_SVG"/>
  </widget>
  <action name="actionMode_Insertion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/pointer.png</normaloff>:/Icones/pointer.png</iconset>
   </property>
   <property name="text">
    <string>Mode &amp;Insertion</string>
   </property>
   <property name="toolTip">
    <string>Permet de déplacer les éléments</string>
   </property>
  </action>
  <action name="actionMode_Edition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/liaison.png</normaloff>:/Icones/liaison.png</iconset>
   </property>
   <property name="text">
    <string>Mode &amp;Edition</string>
   </property>
   <property name="toolTip">
    <string>Permet de créer une hiérarchie entre les éléments</string>
   </property>
  </action>
  <action name="actionIteration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/vignette_boucle.png</normaloff>:/Icones/vignette_boucle.png</iconset>
   </property>
   <property name="text">
    <string>Itération</string>
   </property>
   <property name="toolTip">
    <string>Permet d'insérer une itération</string>
   </property>
  </action>
  <action name="actionCondition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/vignette_condition.png</normaloff>:/Icones/vignette_condition.png</iconset>
   </property>
   <property name="text">
    <string>condition</string>
   </property>
   <property name="toolTip">
    <string>Permet d'insérer une condition</string>
   </property>
  </action>
  <action name="actionAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/vignette_action.png</normaloff>:/Icones/vignette_action.png</iconset>
   </property>
   <property name="text">
    <string>Action</string>
   </property>
   <property name="toolTip">
    <string>Permet d'insérer une action</string>
   </property>
  </action>
  <action name="actionProcedure">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/vignette_procedure.png</normaloff>:/Icones/vignette_procedure.png</iconset>
   </property>
   <property name="text">
    <string>procedure</string>
   </property>
   <property name="toolTip">
    <string>Permet d'insérer un appel de procedure</string>
   </property>
  </action>
  <action name="actionConditionMultiple">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/vignette_conditionMultiple.png</normaloff>:/Icones/vignette_conditionMultiple.png</iconset>
   </property>
   <property name="text">
    <string>ConditionMultiple</string>
   </property>
   <property name="toolTip">
    <string>Permet d'insérer une condition multiple</string>
   </property>
  </action>
  <action name="actionSortie">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/vignette_sortie.png</normaloff>:/Icones/vignette_sortie.png</iconset>
   </property>
   <property name="text">
    <string>Sortie</string>
   </property>
   <property name="toolTip">
    <string>Permet d'insérer une action de sortie</string>
   </property>
  </action>
  <action name="actionNouveau">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/newFile.png</normaloff>:/Icones/newFile.png</iconset>
   </property>
   <property name="text">
    <string>Nouveau</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/ouvrir.png</normaloff>:/Icones/ouvrir.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/icone_sauvegarder.png</normaloff>:/Icones/icone_sauvegarder.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExporter_vers_une_image">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/exporter_Image.png</normaloff>:/Icones/exporter_Image.png</iconset>
   </property>
   <property name="text">
    <string>Exporter en PNG</string>
   </property>
   <property name="toolTip">
    <string>Permet d'exporter vers le format de fichier PNG</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+I</string>
   </property>
  </action>
  <action name="actionFermer_l_onglet">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/fermer_Onglet.png</normaloff>:/Icones/fermer_Onglet.png</iconset>
   </property>
   <property name="text">
    <string>Fermer l'onglet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/fermer_Application.png</normaloff>:/Icones/fermer_Application.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRedimensionner_l_Algorithme">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/icone_resize.png</normaloff>:/Icones/icone_resize.png</iconset>
   </property>
   <property name="text">
    <string>Redimensionner la feuille de dessin</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionV_rifier_les_cycles">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Rechercher Incohérences</string>
   </property>
  </action>
  <action name="actionVers_Ada">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/ada.jpg</normaloff>:/Icones/ada.jpg</iconset>
   </property>
   <property name="text">
    <string>Vers Ada</string>
   </property>
  </action>
  <action name="actionVers_Java">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/java.png</normaloff>:/Icones/java.png</iconset>
   </property>
   <property name="text">
    <string>Vers Java</string>
   </property>
  </action>
  <action name="actionVers_C_C">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/c.png</normaloff>:/Icones/c.png</iconset>
   </property>
   <property name="text">
    <string>Vers C/C++</string>
   </property>
  </action>
  <action name="actionA_propos_de_Tabula_Rasa">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/icone_Application.png</normaloff>:/Icones/icone_Application.png</iconset>
   </property>
   <property name="text">
    <string>A propos de Tabula Rasa</string>
   </property>
  </action>
  <action name="actionA_propos_de_Qt">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/qt.png</normaloff>:/Icones/qt.png</iconset>
   </property>
   <property name="text">
    <string>A propos de Qt</string>
   </property>
  </action>
  <action name="actionManuel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Manuel</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionRenommer_l_algorithme">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/icone_rename.png</normaloff>:/Icones/icone_rename.png</iconset>
   </property>
   <property name="text">
    <string>Renommer l'algorithme</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionImprimer">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/icone_imprimante.png</normaloff>:/Icones/icone_imprimante.png</iconset>
   </property>
   <property name="text">
    <string>Imprimer</string>
   </property>
   <property name="toolTip">
    <string>Imprimer l'algorithme</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionTout_s_lectionner">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/select_all.png</normaloff>:/Icones/select_all.png</iconset>
   </property>
   <property name="text">
    <string>Tout sélectionner</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionExporter_SVG">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/export_svg.png</normaloff>:/Icones/export_svg.png</iconset>
   </property>
   <property name="text">
    <string>Exporter en SVG</string>
   </property>
   <property name="toolTip">
    <string>Permet d'exporter vers le format de fichier SVG</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionExporter_en_PDF">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/export_pdf.png</normaloff>:/Icones/export_pdf.png</iconset>
   </property>
   <property name="text">
    <string>Exporter en PDF</string>
   </property>
   <property name="toolTip">
    <string>Permet d'exporter vers le format de fichier pdf</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+P</string>
   </property>
  </action>
  <action name="actionSauvegarder_sous">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/icone_sauvegarder.png</normaloff>:/Icones/icone_sauvegarder.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer sous</string>
   </property>
   <property name="toolTip">
    <string>Sauvegarder l'agorithme vers un nouveau fichier</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSupprimer_la_s_lection">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/delete.png</normaloff>:/Icones/delete.png</iconset>
   </property>
   <property name="text">
    <string>Supprimer la sélection</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionCopier_la_s_lection">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/copy.png</normaloff>:/Icones/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copier la sélection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionColler_la_s_lection">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/paste.png</normaloff>:/Icones/paste.png</iconset>
   </property>
   <property name="text">
    <string>Coller la sélection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Ressources/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
